data.rb

```diff
@@ -0,0 +1,27 @@
+require /Users/kirillgavris/Desktop/Library/autoload.rb
+
+def save_library_data(file_name)
+    file_name = "Library.yml"
+    data = {
+      authors: YAML.dump(@authors),
+      books: YAML.dump(@books),
+      readers: YAML.dump(@readers),
+      orders: YAML.dump(@orders)
+    }
+    file = File.new(file_name, 'w')
+    file.write(YAML.dump data)
+  end
+
+  def load_library_data(file_name)
+    file_name = "load.yml"
+    if File.exist?(file_name)
+      file = File.open(file_name)
```

Comment:
You must explicitly close file after loading all data from it. Otherwise the file descriptor on your machine will drain more memory.

Reply:
Closed